### ISSUE TITLE
Strange warnings: Cannot find source file

### ISSUE THREAD

**Issue by Vargentum** [fahad19/proppy#14]
Hey, thanks for cool lib. It implements nice separation of concerns approach!

I'm using proppy + proppy-preact. and I received strange warnings when I'm trying to import proppy's utils into my project. Everything works fine, but these warnings trash my console and I want to remove them. 

Here is an example:

```WARNING in ./node_modules/proppy/lib/compose.js
(Emitted value instead of an instance of Error) Cannot find source file '../src/compose.ts': Error: Can't resolve '../src/compose.ts' in '/Users/user/project/node_modules/proppy/lib'
 @ ./node_modules/proppy/lib/index.js 3:16-36
 @ ./src/components/pages/calculator/Calculator.js
 @ ./src/components/pages/index.js
 @ ./src/components/app.js
 @ ./src/index.js
 @ multi (webpack)-dev-server/client?http://localhost:8080 webpack/hot/dev-server ./index.js```

**Comment by fahad19** [comment]
Thanks for the positive feedback, @Vargentum! 💖 

Could you kindly share rest of your code too? Like how you are importing it. That would really help me understand.

An example on CodeSandbox would be helpful too.

For example, the playground with proppy + proppy-react is here: https://codesandbox.io/s/github/fahad19/proppy/tree/master/examples/react-playground

**Comment by fahad19** [comment]
I am assuming you are using TypeScript. 

And it could be because of this line in `.npmignore`, where we do not push the `src` directory to npm: https://github.com/fahad19/proppy/blob/master/packages/proppy/.npmignore#L3

If you want to send a PR that removes the `src` line from `.npmignore` of all packages, that may solve it.

**Comment by Vargentum** [comment]
Sure. Here are the imports:
```import { compose, withState, withHandlers, didSubscribe } from 'proppy';
import { attach } from 'proppy-preact'```

I'm not using TypeScript. 
I also run build to exclude issues with webpack-dev-server. Warnings still remain, so it's not a source.
Could it be related with webpack resolving modules setup? I have following config:
```...
resolve: {
         extensions: ['.jsx', '.js', '.json', '.styl'],
	modules: [
		path.resolve(__dirname, "src/lib"),
		path.resolve(__dirname, "node_modules"),
		'node_modules'
	],
	alias: {
		components: path.resolve(__dirname, "src/components"),    // used for tests
		style: path.resolve(__dirname, "src/style"),
		'react': 'preact-compat',
		'react-dom': 'preact-compat'
	}
},
...```

**Comment by fahad19** [comment]
I have just published v1.2.2 which has the `src` directory published to npm too. Could you please try again? It should solve your warnings issue.

**Comment by Vargentum** [comment]
Yes it helps. 
Thanks.
